extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -414,6 +432,24 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     this.qqid_ = null;
   }
 
+  /** @override */
+  layoutCallback() {
+    const superReturnValue = super.layoutCallback();
+    this.refreshManager_ = this.refreshManager_ || new RefreshManager(this, {
```

Comment:
Make sure to add check that we do not allow refresh if SRA is enabled.  Ideally we would also report this to the pub